pkg/api/server/v1alpha2/results_test.go

```diff
@@ -325,7 +325,9 @@ func TestListResults(t *testing.T) {
 
 	parent := "foo"
 	results := make([]*pb.Result, 0, 5)
+
 	for i := 1; i <= cap(results); i++ {
+		fakeClock.Advance(1)
```

Comment:
super nit: Prefer using a qualified unit so it's clear how much we're incrementing (e.g. `1*time.Second`). 

Since Duration's are nanoseconds, incrementing by such a small number may cause problems if we're not careful about preserving the precision (e.g. if unix time is ever introduced anywhere).

Reply:
Good point, I prefer the explicitness of `time.Second` over `1`.